packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -60,4 +60,10 @@ export interface ICheckboxProps extends React.HTMLProps<HTMLElement | HTMLInputE
    * precedence over the later.
    */
   inputProps?: React.HTMLProps<HTMLElement | HTMLInputElement>;
+
+  /**
+   * Allows you to change the position of the checkbox to the right side of the label
```

Comment:
"Allows you to set the checkbox to be at the before (start) or after (end) the label."